app/src/main/java/pro/itshark/moneysplitter/presentation/addUsers/UsersAdapter.kt

```diff
@@ -0,0 +1,46 @@
+package pro.itshark.moneysplitter.presentation.addUsers
+
+import android.arch.lifecycle.LifecycleOwner
+import android.arch.lifecycle.Observer
+import pro.itshark.moneysplitter.R
+import pro.itshark.moneysplitter.common.RecyclerBaseAdapter
+import pro.itshark.moneysplitter.presentation.userProfile.UserModel
+
+class UsersAdapter(private val owner: LifecycleOwner, var items: List<UserModel>) : RecyclerBaseAdapter() {
+
+    private val selectedUsers: MutableMap<Long, UserModel> = mutableMapOf()
+
+    private val stateViewModelObserver = Observer<UserItemViewModel.State> { state ->
+        state?.let {
+            when (state) {
+                is UserItemViewModel.State.Default -> {
```

Comment:
empty?

Reply:
Yes, in present moment we don't should to change anything in default state.